tests/cubegeom.c

```diff
@@ -55,7 +60,11 @@ int main(int argc, char *argv[])
         printf("Unable to set video mode: %s\n", SDL_GetError());
         return 1;
     }
-    
+
+#if USE_REGAL
+    RegalMakeCurrent((void*)1);
```

Comment:
Is the parameter not actually used, and I guess the 1 is because a 0 would be seen as an error?

Is it not possible to just not call this at all?

Reply:
Well maybe not SDL implementation in fact. It is not supposed to know that GL backend is Regal

Reply:
After a second look in Regal code, it looks like Regal is able to support several independent contexts. The "1" is simply the first context, and it have to be somehow "set as active", in particular for Regal to do some initializations.
Not clear to me who should do this except the Regal user code. SDL or maybe EGL too, but they would both need to know that they are using Regal.